packages/cozy-codeshifts/src/utils/add-import.js

```diff
@@ -0,0 +1,16 @@
+const addImport = (root, j, options) => {
```

Comment:
Can you add a comment with a before / after example? I think this would be easier so figure out what it does than by reading the code.

Reply:
Thanks!